sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj

```diff
@@ -8,31 +8,35 @@
     <ApiCompatVersion>1.2.0</ApiCompatVersion>
     <PackageTags>Microsoft Azure Application Configuration;Data;AppConfig;$(PackageCommonTags)</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <NoWarn>$(NoWarn);3021</NoWarn>
+    <NoWarn>$(NoWarn);3021;AZC0007;AZC0006</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>$(DefineConstants);EXPERIMENTAL</DefineConstants>
```

Comment:
Interesting...  what causes us to need the `Experimental` constant?

Reply:
It was initially copied from the other DPG libraries (e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Azure.IoT.DeviceUpdate.csproj#L12), but I have just updated it and after removing the experimental dependency, the code runs fine, thanks!